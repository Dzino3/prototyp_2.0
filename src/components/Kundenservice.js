import React from 'react'
import './style/kundenserviceStyle.css';

function Kundenservice() {
  return (
    <>
    <details className="custom-select">
      <summary className="radios">
        <input className='kundenInput' type="radio" name="item" id="default" title="Customer support" checked />
        <input className='kundenInput' type="radio" name="item" id="item1" title="FAQ" />
        <input className='kundenInput' type="radio" name="item" id="item2" title="Service" />
        <input className='kundenInput' type="radio" name="item" id="item3" title="Newsletter" />
        <input className='kundenInput' type="radio" name="item" id="item4" title="Privacy policy" />
      </summary>
      <ul className="list">
        <li className='kundenLi'>
        <label for="item1">
          <a href='https://www.deere.de/de/ersatzteile-und-service/ersatzteile/suche-nach-teilen-h%C3%A4ufig-gestellte-fragen/' target="blank">FAQ</a>
          <span></span>
        </label>
        </li>
        <li className='kundenLi'>
          <label for="item2">
            <a target="blank" href='https://www.deere.de/de/ersatzteile-und-service/service/'>Service</a>
            </label>
        </li>
        <li className='kundenLi'>
          <label for="item3">
            <a target="blank" href='https://www.deere.de/de/forms/digital-news-service/'>Newsletter</a>
          </label>
        </li>
        <li className='kundenLi'>
          <label for="item4">
            <a target="blank" href='https://www.deere.de/de/datenschutzerkl%C3%A4rung/'>Privacy and Data</a>
            </label>
        </li>
        </ul>
    </details>
    </>
  )
}

export default Kundenservice