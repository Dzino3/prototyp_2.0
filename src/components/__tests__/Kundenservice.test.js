import Kundenservice from '../Kundenservice';
import React from "react";
import renderer from 'react-test-renderer';


describe(`Kundenservice Rendering`, () => {

    test('should render Kundenservice', function () {
        let tree = renderer.create(<Kundenservice/>);

        expect(tree.toJSON()).toMatchSnapshot();
    });
})