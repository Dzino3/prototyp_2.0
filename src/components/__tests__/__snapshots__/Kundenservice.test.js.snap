// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Kundenservice Rendering should render Kundenservice 1`] = `
<details
  className="custom-select"
>
  <summary
    className="radios"
  >
    <input
      checked={true}
      className="kundenInput"
      id="default"
      name="item"
      title="Customer support"
      type="radio"
    />
    <input
      className="kundenInput"
      id="item1"
      name="item"
      title="FAQ"
      type="radio"
    />
    <input
      className="kundenInput"
      id="item2"
      name="item"
      title="Service"
      type="radio"
    />
    <input
      className="kundenInput"
      id="item3"
      name="item"
      title="Newsletter"
      type="radio"
    />
    <input
      className="kundenInput"
      id="item4"
      name="item"
      title="Privacy policy"
      type="radio"
    />
  </summary>
  <ul
    className="list"
  >
    <li
      className="kundenLi"
    >
      <label
        for="item1"
      >
        <a
          href="https://www.deere.de/de/ersatzteile-und-service/ersatzteile/suche-nach-teilen-h%C3%A4ufig-gestellte-fragen/"
          target="blank"
        >
          FAQ
        </a>
        <span />
      </label>
    </li>
    <li
      className="kundenLi"
    >
      <label
        for="item2"
      >
        <a
          href="https://www.deere.de/de/ersatzteile-und-service/service/"
          target="blank"
        >
          Service
        </a>
      </label>
    </li>
    <li
      className="kundenLi"
    >
      <label
        for="item3"
      >
        <a
          href="https://www.deere.de/de/forms/digital-news-service/"
          target="blank"
        >
          Newsletter
        </a>
      </label>
    </li>
    <li
      className="kundenLi"
    >
      <label
        for="item4"
      >
        <a
          href="https://www.deere.de/de/datenschutzerkl%C3%A4rung/"
          target="blank"
        >
          Privacy and Data
        </a>
      </label>
    </li>
  </ul>
</details>
`;
